environments/environment.py

```diff
@@ -182,7 +182,8 @@ def callback(ch, method, properties, body):
             try:
                 decoded = json.loads(body)
                 f1 = e1.Assert("(ServerFact \"" + str(decoded[SERVERID]) + "\" " + str(decoded['cpu'])
-                               + " " + str(decoded['mem']) + ")")
+                               + " " + str(decoded['mem']) + " " + str(decoded['hdd']) + " " + str(decoded['net'])
```

Comment:
Any PR should include the new functionalities, any modification in documentation if it is required and the corresponding unit test to cover the functionality.